cache-infinispan/pom.xml

```diff
@@ -23,6 +22,16 @@
         <groupId>org.infinispan</groupId>
         <artifactId>infinispan-core</artifactId>
     </dependency>
+    
+    <dependency>
```

Comment:
Why these dependencies?

Reply:
Notice that they were moved from runtime to cache.  They are transitive dependencies of infinispan.